* When quarto is run unbundled/QUARTO_DEBUG it checks to see if it needs 
be reconfigured.  It checks this by checking if two configuration
files are in agrement and checking the hash of some files.
If not equal it start the configuration script.
I added the console.log to know if something was missed.
--- a/src/quarto.ts
+++ b/src/quarto.ts
@@ -56,8 +56,7 @@ export async function quarto(
     const installed = readInstalledDevConfig();
     const source = readSourceDevConfig();
     if (installed == null || !devConfigsEqual(installed, source)) {
-      await reconfigureQuarto(installed, source);
-      Deno.exit(1);
+      console.log("Quarto thinks it needs reconfigured");
     }
   }
 
